ratpack-core/src/main/java/ratpack/http/client/RequestSpec.java

```diff
@@ -117,6 +117,12 @@ default RequestSpec sslContext(Factory<SSLContext> factory) throws Exception {
 
   URI getUrl();
 
+  default RequestSpec connectTimeoutSeconds(int seconds) {
```

Comment:
I think we probably want some kind of sensible default here if this isn't set explicitly. Anyone know of any precedent for the default?